src/main/java/uk/gov/hmcts/reform/sscs/bulkscan/validators/FormTypeValidator.java

```diff
@@ -48,32 +55,38 @@ public CaseResponse validate(String caseId, ExceptionRecord exceptionRecord) {
                 .status(getValidationStatus(errors, null)).build();
         }
 
-        try {
-            FormType formType = FormType.getById(formTypeData);
+        FormType formType = FormType.getById(formTypeData);
 
-            log.info("Validating against formType {}", formType);
+        log.info("Validating against formType {}", formType);
 
-            if (formType != null && formType.equals(FormType.SSCS2)) {
-                sscs2Schema.validate(new JSONObject(build(exceptionRecord.getOcrDataFields())));
-            } else if (formType != null && formType.equals(FormType.SSCS5)) {
-                sscs5Schema.validate(new JSONObject(build(exceptionRecord.getOcrDataFields())));
-            } else if (formType != null && (formType.equals(FormType.SSCS1U) || formType.equals(FormType.SSCS1)
-                || formType.equals(FormType.SSCS1PE) || formType.equals(FormType.SSCS1PEU))) {
-                sscs1Schema.validate(new JSONObject(build(exceptionRecord.getOcrDataFields())));
-            }
-        } catch (ValidationException ex) {
-            log.error("Validation failed: {}", ex.getAllMessages());
+        JsonNode jsonNode = mapper.valueToTree(build(exceptionRecord.getOcrDataFields()));
+        Set<ValidationMessage> validationErrors = null;
+
+        if (formType != null && formType.equals(FormType.SSCS2)) {
```

Comment:
maybe could've used a switch statement here with an external if statement for the null check